Docs/Configuration.tex

```diff
@@ -2665,10 +2665,10 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
 
   \emph{Note 1}: This option resolves compatibility issues with Wi-Fi, Ethernet and
   Thunderbolt devices when AppleVTD is enabled on systems where the native DMAR table
-  contains one or more Reserved Memory Regions.
+  contains one or more Reserved Memory Regions and iGPU is enabled and more than
+  16GB memory is installed.
```

Comment:
`16 GB`; there are several instances in the documentation where there is one space between the numeric value and unit.

Reply:
Change has been made.